client/src/components/Dashboard/flights/flights.js

```diff
@@ -36,31 +47,73 @@ const Flights = () => {
     setOpen(false);
   };
 
-  const onSubmit = async (e) => {
+  const onSubmit = (e) => {
     e.preventDefault();
 
-    const { flightNumber, flightTime, airline, departureAirport, departureTerminal, departureCity, departureGate, arrivalAirport, arrivalTerminal, arrivalCity, arrivalGate, bookingReference, isOutbound } = flight;
+    const { flightNumber, departureTime, departureDate, airline, departureAirport, departureTerminal, departureCity, departureGate, arrivalAirport, arrivalTerminal, arrivalCity, arrivalGate, bookingReference, isOutbound } = flight;
 
-    const newFlight = { flightNumber, flightTime, airline, departureAirport, departureTerminal, departureCity, departureGate, arrivalAirport, arrivalTerminal, arrivalCity, arrivalGate, bookingReference, isOutbound };
+    const newFlight = { flightNumber, departureTime, departureDate,  airline, departureAirport, departureTerminal, departureCity, departureGate, arrivalAirport, arrivalTerminal, arrivalCity, arrivalGate, bookingReference, isOutbound };
 
-    await axios.post("http://localhost:8000/dashboard/flights/", newFlight).then(() => {
+    api.post("/", newFlight).then(res => {
       handleClose();
       window.location = "/";
     });
   };
 
-  return (
-    <div className="Flights">
-      <AddFlight
-        open={open}
-        handleOpen={handleOpen}
-        handleClose={handleClose}
-        handleChange={handleChange}
-        flight={flight}
-        onSubmit={onSubmit}
-      />
-    </div>
-  );
+  useEffect(() => {
+    api.get("/").then(res => {
+      const outbound = res.data.outbound    
+      const inbound = res.data.inbound
+      setOutboundFlight(outbound);
+      setInboundFlight(inbound);
+    });
+  }, [])
+
+  const outboundFlights = [];
+  const inboundFlights = [];
+
+  outboundFlight.forEach((flight) => {
+    outboundFlights.push(<FlightCard outboundFlight={flight} />)
+  })
+
+  inboundFlight.forEach((flight) => {
+    inboundFlights.push(<FlightCard outboundFlight={flight} />)
+  })
+
+  if (outboundFlight.length || inboundFlight.length) {
+    return(
+      <div className="flights-window">
+        <div className="flights-header">
+          <h1>Your flights</h1>
+        </div>
+        <div className="flights-content">
+          <div className="flights-content-outbound">
+            <h1 className="flights-content-subheading">Outbound</h1>
+            {outboundFlight[0] && outboundFlights}
+          </div>
+          <div className="flights-content-inbound">
+            <h1 className="flights-content-subheading">Inbound</h1>
+            {inboundFlight[0] && inboundFlights}
+          </div>
+        </div>
+        <div className="flights-footer">
+          <AddFlight
+            open={open}
+            handleOpen={handleOpen}
+            handleClose={handleClose}
+            handleChange={handleChange}
+            flight={flight}
+            onSubmit={onSubmit}
```

Comment:
Can we please change to handleSubmit as other components will be using that, for consistency